include/oneapi/dpl/pstl/execution_defs.h

```diff
@@ -31,88 +31,20 @@ inline namespace v1
 // 2.4, Sequential execution policy
 class sequenced_policy
```

Comment:
@rarutyun Will we have here ABI breaking change?
We remove methods from public class...

Reply:
Agree